src/betterproto/__init__.py

```diff
@@ -972,6 +975,8 @@ def to_dict(
                     )
                 ):
                     output[cased_name] = value.to_dict(casing, include_default_values)
+                elif self.__raw_get(field_name) != PLACEHOLDER:
```

Comment:
Similar thing here, should be `is not`

Reply:
I was thinking about that but I saw line 540 doing the same comparison